Client.xcodeproj/project.pbxproj

```diff
@@ -9,6 +9,7 @@
 /* Begin PBXBuildFile section */
 		03CCC9181AF05E7300DBF30D /* RelativeDatesTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 03CCC9171AF05E7300DBF30D /* RelativeDatesTests.swift */; };
 		0430A545203B372D00FDF76D /* IntegrationTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0430A544203B372D00FDF76D /* IntegrationTests.swift */; };
+		048B6E4F24AEC7D70087EA70 /* Strings.swift in Sources */ = {isa = PBXBuildFile; fileRef = D35210E01CB2F16600FC5DCB /* Strings.swift */; };
```

Comment:
@noorhashem Don't think we need to modify project.pbxproj, was strings file missing? 🤔

Reply:
it's because I added the Today widget extension to the targets exposed to the strings.swift file because it was out of scope.

Reply:
So this got added to Shared? I just tried to compile without these changes and am able to run the app.

Reply:
I ticked the Today extension in the target membership of the Strings.swift file because I got this error without doing this change : "Strings doesn't have such member"

The shared was already ticked btw.
I will try compiling again without it, weird because it failed with me without it